### PR TITLE
fix: add ask to delete to delete workunits

### PR DESCRIPTION
**Pull request by dehilsterlexis**
#378

### CODE REVIEW

File: src/ecl/eclWatchTree.ts [hunk 1]
```diff
@@ -505,7 +505,15 @@ export class ECLWUNode extends Item<ECLWatchTree> {
...
-        this._wu.delete().then(() => this._tree.refresh());
+        const items: vscode.QuickPickItem[] = [];
+		items.push({label: localize("Yes"), description: `${localize("Delete Workunit")} ${this._wu.Wuid}`});
+		items.push({label: localize("No"), description: localize("Do not delete")});
+
+		vscode.window.showQuickPick(items, {title: localize("Delete Workunit"), canPickMany: false, placeHolder: localize("Choose Yes or No")}).then(selection => {
+			if (!selection || selection.label == localize("No"))
+				return;
+            this._wu.delete().then(() => this._tree.refresh());
+		});
```
Comment (GordonSmith) [inline src/ecl/eclWatchTree.ts:516]: Indentation looks weird

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Comment by GordonSmith** [comment]
@dehilsterlexis Looks like you merged trunk into this branch, instead of rebasing this branch onto upstream/trunk?

**Comment by dehilsterlexis** [comment]
@GordonSmith I tried using the conflict resolution in github and it added that commit.

**Comment by GordonSmith** [comment]
Yes that is what it looked like, I have straightened it out, but in the future just rebase to resolve any conflicts...

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**